input/postgres/explain.go

```diff
@@ -16,8 +16,21 @@ import (
 func RunExplain(server state.Server, inputs []state.PostgresQuerySample, collectionOpts state.CollectionOpts, logger *util.Logger) (outputs []state.PostgresQuerySample) {
 	var samplesByDb = make(map[string]([]state.PostgresQuerySample))
 
+	skip := func(sample state.PostgresQuerySample) bool {
+		monitoredDb := sample.Database == server.Config.GetDbName() || server.Config.DbAllNames || contains(server.Config.DbExtraNames, sample.Database)
```

Comment:
@lfittl hmm, from our conversation earlier, if `sample.Database` is the empty string, this would be `false`, no? And then we'd skip based on the first condition below?

Reply:
Good point. I think we should add `sample.Database == "" || ...` here